orcestra_book/plans/HALO_20240919a_sfc_winds.py

```diff
@@ -0,0 +1,83 @@
+"""generates surface wind speed plots with
+    low wind speed regions identified using ECMWF IFS.
+    Functions copied from sfc_winds.py from orcestra
+    weather briefing repository"""
+
+import easygems.healpix as egh
+import numpy as np
+import healpy as hp
```

Comment:
We dropped the `healpy` support because of its dependency overhead and lack of Windows support. However, in your case `healpix` should provide a drop-in replacement

Reply:
super thanks! :)